### PR TITLE
DDF-5618 upgrade tika to version 1.22

### PR DESCRIPTION
**Pull request by josephthweatt**
#### What does this PR do?
Upgrades Tika to 1.22

Only introduce after adding in the thirdparty change:
https://github.com/codice/thirdparty/pull/39

#### Who is reviewing it? 
@aaronilovici 
@rfding 

#### Ask 2 committers to review/merge the PR and tag them here.

@bdeining
@oconnormi
#### How should this be tested?
Build this branch and test it by ingesting a visio file to ensure that the metadata was extracted.

#### What are the relevant tickets?
Fixes: #DDF-5618

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/yarn.lock [hunk 1]
```diff
@@ -15361,6 +15361,11 @@ lodash@4.13.1:
   version "4.13.1"
   resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.13.1.tgz#83e4b10913f48496d4d16fec4a560af2ee744b68"
 
+lodash@4.17.15, lodash@^4.0.1, lodash@^4.17.12, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15:
```
Comment (shaundmorris) [inline ui/yarn.lock]: @josephthweatt just FYSA, I've been merging a few dependabot PRs that upgrade lodash to 4.17.15 in a few PRs.

Comment (shaundmorris) [reply ui/yarn.lock]: I merged in all the PRs that upgraded lodash to 4.17.15 in their respective package.json files.

**Comment by josephthweatt** [comment]
build now

**Comment by josephthweatt** [comment]
build now

File: ui/yarn.lock (hunk 1 above)
Comment (josephthweatt) [reply ui/yarn.lock]: lodash changes removed after rebasing

**Comment by mdang8** [comment]
✅ Hero success! Ingested a VSDX file with the Tika transformer and verified that it shows up in search with the preview showing the correct information.

**Pull request merged**